mlrun/model_monitoring/stream_processing.py

```diff
@@ -259,39 +258,13 @@ def apply_parquet_target():
 
         # controller branch
         def apply_push_controller_stream(stream_uri: str):
-            if stream_uri.startswith("v3io://"):
-                graph.add_step(
-                    ">>",
-                    "controller_stream_v3io",
-                    path=stream_uri,
-                    sharding_func=ControllerEvent.ENDPOINT_ID,
-                    access_key=self.v3io_access_key,
-                    after="ForwardNOP",
-                )
-            elif stream_uri.startswith("kafka://"):
-                topic, brokers = parse_kafka_url(stream_uri)
-                logger.info(
-                    "Controller stream uri for kafka",
-                    stream_uri=stream_uri,
-                    topic=topic,
-                    brokers=brokers,
-                )
-                if isinstance(brokers, list):
-                    path = f"kafka://{brokers[0]}/{topic}"
-                elif isinstance(brokers, str):
-                    path = f"kafka://{brokers}/{topic}"
-                else:
-                    raise mlrun.errors.MLRunInvalidArgumentError(
-                        "Brokers must be a list or str check controller stream uri"
-                    )
-                graph.add_step(
-                    ">>",
-                    "controller_stream_kafka",
-                    path=path,
-                    kafka_brokers=brokers,
-                    _sharding_func=ControllerEvent.ENDPOINT_ID,
-                    after="ForwardNOP",
-                )
+            graph.add_step(
+                ">>",
+                "controller_stream",
+                path=stream_uri,
```

Comment:
Ideally, this function should not get the obsolete `stream_uri` argument.
When looking behind the scenes, this argument is initialized by calling:
`get_stream_path`
Which doesn't cover all the functionality provided by the profiles. For example, in Kafka, it takes only the first broker, even when there is more than one.

`mlrun.model_monitoring.helpers.get_output_stream` is the newer version that utilizes the full features of the stream's profile.